upf/upf_pfcp_api.c

```diff
@@ -1908,6 +1916,12 @@ handle_update_urr (upf_session_t * sx, pfcp_update_urr_t * update_urr,
 	update->time_quota.period = urr->time_quota;
 	update->time_quota.base = update->start_time;
       }
+    if (ISSET_BIT (urr->grp.fields, UPDATE_URR_QUOTA_VALIDITY_TIME))
+      {
+	update->update_flags |= PFCP_URR_UPDATE_QUOTA_VALIDITY_TIME;
+	update->quota_validity_time.period = urr->quota_validity_time;
+	update->quota_validity_time.base = now;
+      }
```

Comment:
wrong order, Quota Validity Time should go after Dropped DL
Traffic Threshold and before Monitoring Time to keep it in sync with TS 29.244.